### ISSUE TITLE
No lr_scheduler in full_finetune_distributed

### ISSUE THREAD

**Issue by jxmsML** [pytorch/torchtune#952]
Learningrate scheduler isn't set up in the  full_finetune_distributed training loop, only exists in the lora finetuning.

**Comment by ebsmothers** [comment]
Hi @jxmsML thanks for pointing this out. We do only support fixed learning rate in the full finetune recipes, but I agree we should support schedulers there like we do in LoRA. There's not really a technical blocker for this, so if you're interested to open a PR for the feature we'd love to have it. Otherwise we hope to add it soon!

**Comment by RdoubleA** [comment]
Duplicated in #1308
